cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
Is this file needed?

Reply:
Yes, this imports commands from command.js file.